models/CookFONLLPythiaPred.C

```diff
@@ -31,12 +32,12 @@ enum { // options for re-weight
   kAccurate // correct each b-hadron contribution indipendently (modify also the pT dependence)
 };
 
-void CookFONLLPythiaPred(std::string inFileNameMin = "DfromB_FONLLminPythia8_FFppbar_yDcut_pp502TeV.root", // min FONLL predictions
-                          std::string inFileNameCent = "DfromB_FONLLcentPythia8_FFppbar_yDcut_pp502TeV.root", // central FONLL predictions
-                          std::string inFileNameMax = "DfromB_FONLLmaxPythia8_FFppbar_yDcut_pp502TeV.root",  // max FONLL predictions
-                          std::string outFileName = "DmesonLcPredictions_502TeV_y05_FFppbar_BRPDGmix_SepContr.root",
+void CookFONLLPythiaPred(std::string inFileNameMin = "DfromB_FONLLminEvtGen_FFppbar_yDcut_pp502TeV.root", // min FONLL predictions
+                          std::string inFileNameCent = "DfromB_FONLLcentEvtGen_FFppbar_yDcut_pp502TeV.root", // central FONLL predictions
+                          std::string inFileNameMax = "DfromB_FONLLmaxEvtGen_FFppbar_yDcut_pp502TeV.root",  // max FONLL predictions
+                          std::string outFileName = "DmesonLcPredictions_502TeV_y05_EvtGen_FFee_BRPDGmix_SepContr.root",
                           int brOpt = kBRPDGmix,
-                          int ffOpt = kFFppbar,
+                          int ffOpt = kFFee,
```

Comment:
I would set here the "default" configuration used for the D2H analysis.

Reply:
Ciao Fabio, yes this could be a good idea indeed (especially for the AP version). Could you do that?

Reply:
Done